docs/schemas/core/IfcControlExtension/Entities/IfcPerformanceHistory.md

```diff
@@ -1,13 +1,13 @@
 # IfcPerformanceHistory
 
-_IfcPerformanceHistory_ is used to document the actual performance of an occurrence instance over time. It includes machine-measured data from building automation systems and human-specified data such as task and resource usage. The data may represent actual conditions, predictions, or simulations.
+_IfcPerformanceHistory_ is used to document the actual performance of an occurrence instance over time. It includes machine-measured data from building automation systems and human-specified data such as task and resource usage. The data may represent actual conditions, predictions, or simulations.<!-- end of definition -->
```

Comment:
My preference would have been to have the comment on a new line. Now it's rather hard to overlook.

```suggestion
_IfcPerformanceHistory_ is used to document the actual performance of an occurrence instance over time. It includes machine-measured data from building automation systems and human-specified data such as task and resource usage. The data may represent actual conditions, predictions, or simulations.

<!-- end of definition -->
```

Reply:
Sometimes, the marker is placed between two sentences of the same paragraph. I didn't want to split the paragraph and affect the existing IFC definitions. If you say we can do that, I'm also in favour of line break before the marker.

Reply:
> Sometimes, the marker is placed between two sentences of the same paragraph

I would say that is actually a clear indication that there should have been a paragraph break. I don't think adding whitespace in general would be seen as something influential. I would say go for it!